src/App.jsx

```diff
@@ -1,15 +1,22 @@
+import 'bootstrap'
+import './App.css'
+import JobSearch from "./components/JobSearch/index.jsx";
+import {useState} from "react";
 import JobListings from "./components/JobListings/index.jsx";
-import Nav from "./components/Navbar/index.jsx";
-import Footer from "./components/Footer/index.jsx";
 
 function App() {
+    const [query, setQuery] = useState('http://0.0.0.0:8080/jobs/recent')
 
-  return (
-    <>
-      <Nav/>
-      <JobListings />
-      <Footer/>
-    </>
-  )
+    const [header, setHeader] = useState('Most recent jobs')
+    const [viewButton, setViewButton] = useState('View all jobs ->')
+
```

Comment:
This is something we'll talk about after the sprint review, but the fact that we have three bits of state that _will always be changed together_ suggests that we could be doing better here, like have one state variable that's an object or the key of an object that contains all the data we need.